.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - release
+  pull_request:
+
+jobs:
+  formatting:
+    name: Formatting and static analysis
+    runs-on: 'ubuntu-20.04'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+      - run: python -m pip install --upgrade pip
+      - run: python -m pip install -r requirements/ci.txt
+      - run: tox -e static
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Apply automatic formatting
+
+  tests:
+    name: Tests
+    needs: formatting
+    strategy:
+      matrix:
+
+        os: ['ubuntu-20.04']
+        python:
+          - version: '3.9'
+            tox-env: 'py39'
+          - version: '3.10'
+            tox-env: 'py310'
+          - version: '3.11'
+            tox-env: 'py311'
```

Comment:
Four lines above are added on top of the template updates.

Reply:
I feel having to provide the tox env is a way to mess up. Could we have a default for the tox env, computed from the Python version? Then only someone who really needs a different one has to bother with this.

In the copier template, not here, I mean.

Reply:
It seems like it will need two steps to do that,
1. Create an environment variable with manipulated string via bash command, 
    i.e. `run: echo ENV_VAR='${{ inputs.var }}' >> $GITHUB_ENV`
2. Use that environment variable as a github action variable via `${{ env.ENV_VAR }}`.

` ENV_VAR='${{ inputs.var }}' ` can be replaced with proper manipulation.

Reply:
Manipulating variables in the template is much easier than in the GitHub action itself. 
It will not be written or updated so often anyways, no...?

Reply:
See my comment on Slack: Can't we determine the Python version from the tox env? It is tied to it anyway.

Reply:
Yes, if you use `tox -e test` it should use the only python version the environment has,
but if you specify a python version under the [testenv], it might use that one instead.
I wasn't sure if that will be the case in other repositories.

And I'm not sure which one tox will use if there are multiple python versions available in the environment...

Reply:
Or if we just do `tox -e py39`, that should just use the python3.9.
But you still need the Python version in the workflow, to set the environment before using tox.
Otherwise, tox can't find the python... as far as I know...?